views.py

```diff
@@ -622,4 +622,42 @@ def report_authors(request):
 
     template = 'reporting/report_authors.html'
 
-    return render(request, template, context)
\ No newline at end of file
+    return render(request, template, context)
+
+
+@is_repository_manager
+def report_preprints_metrics(request):
+    start_date, end_date = logic.get_metrics_start_end(
```

Comment:
It is a bit of a shame that the new form doesn't handle this :(

Reply:
Form now handles this... quite well I may add.